examples/sprites/sprite_sheet_loader.rs

```diff
@@ -0,0 +1,102 @@
+use amethyst::renderer::{Sprite, SpriteSheet};
+
+use sprite;
+
+/// Loads a sprite sheet from the assets folder.
+///
+/// # Parameters:
+///
+/// * `index`: Index of the sprite sheet's texture in the `MaterialTextureSet`.
+/// * `definition`: Definition of the sprite layout on the sprite sheet.
+pub fn load(index: usize, definition: &sprite::SpriteSheetDefinition) -> SpriteSheet {
```

Comment:
Shouldn't that be directly into the engine? It seems pretty inconvenient to code this each time you make a new project.

Reply:
the `SpriteSheetDefinition` is user defined, based on https://github.com/amethyst/amethyst/issues/634#issuecomment-380568448, so it has to be outside

Reply:
okay, I'll make myself a predefined one then ;)